requirements.txt

```diff
@@ -0,0 +1,9 @@
+asgiref==3.5.2
+Django==4.1
+psycopg2
+pytz==2022.2.1
+sqlparse==0.4.2
+tzdata==2022.2
+djangorestframework
+pylint==3.3.2
+pylint-django
```

Comment:
You must specify the version of all dependencies.
Update this:
- psycopg2
- djangorestframework
- pylint-django

Reply:
I am already updated it in a 
previous pr